operator/operator.make

```diff
@@ -51,12 +54,25 @@ operator-generate-catalog: _work/bin/operator-sdk-$(OPERATOR_SDK_VERSION) _work/
 	@sed -i -e 's;X.Y.Z;$(VERSION);g' -e 's;X.Y;$(SHORT_VERSION);g' $(CATALOG_DIR)/$(VERSION)/pmem-csi-operator.clusterserviceversion.yaml
 	$(MAKE) operator-clean-crd
 
+# Generate OLM bundle. OperatorHub/OLM still does not support bundle format
+# but soon it will move from 'packagemanifests' to 'bundles'.
 operator-generate-bundle: _work/bin/operator-sdk-$(OPERATOR_SDK_VERSION) _work/kustomize operator-generate-crd
 	@echo "Generating operator bundle ..."
 	@_work/kustomize build --load_restrictor=none $(MANIFESTS_DIR) | $< generate bundle  --version=$(VERSION) \
         --kustomize-dir=$(MANIFESTS_DIR) --output-dir=$(BUNDLE_DIR)
 	@sed -i -e 's;X.Y.Z;$(VERSION);g' -e 's;X.Y;$(SHORT_VERSION);g' $(CATALOG_DIR)/$(VERSION)/pmem-csi-operator.clusterserviceversion.yaml
 
+# Test catalog ensures the generated manifests are valid to start the operator by OLM.
+# We first install OLM into our test cluster and ask to run the operator using the manifests
+operator-test-catalog: _work/bin/operator-sdk-$(OPERATOR_SDK_VERSION) operator-generate-catalog start
```

Comment:
Instead of adding new make targets, perhaps write an E2E test which does this? Then it automatically runs in our CI, with results captured in the JUnit file and thus visible in Jenkins?